LayoutTests/editing/find/cocoa/find-and-replace-adjacent-words.html

```diff
@@ -11,22 +12,30 @@
     editor.textContent = "appleappleapple";
 }
 
+Markup.waitUntilDone();
 Markup.description("Verifies that find and replace can be used to replace adjacent words in an editable area. This test requires WebKitTestRunner.");
 
-testRunner.findStringMatchesInPage("apple", []);
-testRunner.replaceFindMatchesAtIndices([0, 1, 2], "", false);
-Markup.dump("editor", "After replacing 'apple' with an empty string");
+onload = async () => {
+    testRunner.findStringMatchesInPage("apple", []);
```

Comment:
Why not make findStringMatchesInPage return a Promise<undefined> and just await it instead of awaiting ensurePresentationUpdate?

Reply:
I think I could do this, but the C API would require me to create a page find client, so this seemed easier.